README.md

```diff
@@ -11,6 +11,18 @@ in pure vim script with python dependency removed.
 Plug 'prabirshrestha/asyncomplete.vim'
 ```
 
+To enable default mappings use the following setting. This is disabled by default.
```

Comment:
Any reason _not_ to enable this by default to provide a better out-of-the-box experience? I guess most people would want these mappings, and if someone wants something more esoteric, they can opt-out.